### ISSUE TITLE
Bot shows links incorrectly

### ISSUE THREAD

**Issue by wokste** [SIlver--/remindmebot-reddit#15]
The bot isn't showing all links correctly anymore. The problem might be related with spaces in the url and/or with the new reddit layout.

Url: https://www.reddit.com/r/FATErpg/comments/8dw8k1/amber_using_fate/
Appearance: ![screenshot error](https://user-images.githubusercontent.com/833206/39095458-f413990c-4640-11e8-9cd7-a62091bbd139.png)
Reddit layout: new layout
Browser: Firefox
Plugins: UBlock origin, multi-account containers.

**Comment by jewel-andraia** [comment]
Yes, the bot should be using `+` or `%20` instead of literal space ` `  in the links.  More generally, the content of the links should be uri-encoded: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote

**Comment by RulerOf** [comment]
Another example of this issue: https://www.reddit.com/r/DataHoarder/comments/c3mahr/please_stop_posting_photos_of_your_hard_drives/ers3jq9
